packages/bodiless-core/src/Types/ContextMenuTypes.tsx

```diff
@@ -96,10 +96,11 @@ export type IContextMenuProps = {
   ui?: UI;
   isPositionToggled?: boolean;
   renderInTooltip?: boolean;
+  closeForm?: (e: any) => void;
```

Comment:
Can we document this passing around of closeForm a bit?  The flow is a bit intricate and may be hard for someone to follow later on.

Reply:
Done